### PR TITLE
Enable configuration file for each bundle

### PR DESCRIPTION
**Pull request by cmizzi**

### CODE REVIEW

File: EventListener/ConfigurationListener.php [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    /** @var Symfony\Component\HttpKernel\Log\LoggerInterface $logger */
+    private $logger;
+
+    /**
+     * __construct.
+     *
+     * @param Symfony\Component\DependencyInjection\ContainerInterface $container
+     */
+    public function __construct(
+        Symfony\Component\HttpKernel\Config\FileLocator $fileLocator,
```
Comment (core23) [inline EventListener/ConfigurationListener.php]: Please don't use FQNs here

**Comment by core23** [comment]
Can you please say something more and add tests @cmizzi

File: EventListener/ConfigurationListener.php (hunk 1 above)
Comment (greg0ire) [inline EventListener/ConfigurationListener.php]: Use use statements instead.

File: EventListener/ConfigurationListener.php [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+use Symfony\Component\HttpKernel\Event\FilterControllerEvent;
+use Symfony\Component\HttpKernel\Config\FileLocator;
+use Symfony\Component\HttpKernel\Log\LoggerInterface;
+use Symfony\Component\Yaml\Yaml;
+use Sonata\SeoBundle\Seo\SeoPage;
+
+class ConfigurationListener
+{
+    /** @var Sonata\SeoBundle\Seo\SeoPage $sonata */
```
Comment (core23) [inline EventListener/ConfigurationListener.php]: Also no FQNs in PHPdoc please

File: EventListener/ConfigurationListener.php [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+    /** @var Symfony\Component\HttpKernel\Log\LoggerInterface $logger */
+    private $logger;
+
+    /**
+     * __construct.
+     *
+     * @param Symfony\Component\DependencyInjection\ContainerInterface $container
+     */
+    public function __construct(
```
Comment (core23) [inline EventListener/ConfigurationListener.php]: Why the linebreaks?

File: EventListener/ConfigurationListener.php [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    /** @var Symfony\Component\HttpKernel\Config\FileLocator $kernel */
+    private $fileLocator;
+
+    /** @var Symfony\Component\HttpKernel\Log\LoggerInterface $logger */
+    private $logger;
+
+    /**
+     * __construct.
+     *
+     * @param Symfony\Component\DependencyInjection\ContainerInterface $container
```
Comment (core23) [inline EventListener/ConfigurationListener.php]: PHPdoc != method signature

File: EventListener/ConfigurationListener.php [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+        try {
+            $path = $this->fileLocator->locate($namespace);
+        } catch (\InvalidArgumentException $e) {
+            // Nothing to do because there's no SEO file
+            return;
+        }
+
+        try {
+            $config = Yaml::parse($path);
+        } catch (Symfony\Component\Yaml\Exception\ParseException $e) {
```
Comment (core23) [inline EventListener/ConfigurationListener.php]: No FQN!

File: EventListener/ConfigurationListener.php (hunk 3 above)
Comment (greg0ire) [inline EventListener/ConfigurationListener.php]: No linebreaks means 91 characters. It's not over the 120 chars hard limit, but is over the 80 chars soft limit, so it should be accepted IMO.

**Comment by core23** [comment]
Ping @cmizzi

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**